src/vm/mngstdinterfaces.cpp

```diff
@@ -994,8 +994,12 @@ FCIMPL1(Object*, StdMngIEnumerable::GetEnumerator, Object* refThisUNSAFE)
 
     if (retVal == NULL)
     {
+#ifndef FEATURE_CORECLR
```

Comment:
Delete it instead of ifdefing - we have deleted all FEATURE_CORECLR ifdefs in CoreCLR and the code protected by them.